.github/workflows/build.yaml

```diff
@@ -52,3 +52,34 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  wheel-build-cuml:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/wheels-manylinux-build.yml@main
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      repo: rapidsai/cuml
```

Comment:
we can remove this `repo` key from all of the `build.yaml` workflows (similarly to how I did for rmm [here](https://github.com/rapidsai/rmm/commit/7fb45d44c8ca4151581f5c57e413d1aea4b00f3f)).

This key was only ever used because we were using a `workflow_call` event to call this workflow from the https://github.com/rapidsai/actions/ repository for nightly builds.

Now that this file uses `workflow_dispatch` instead of `workflow_call`, this key isn't relevant anymore.

We should make this change for all jobs in this workflow and also all other wheel PRs.